pkg/git/git.go

```diff
@@ -45,10 +46,11 @@ func run(logger *zap.SugaredLogger, dir string, args ...string) (string, error)
 
 // FetchSpec describes how to initialize and fetch from a Git repository.
 type FetchSpec struct {
-	URL      string
-	Revision string
-	Path     string
-	Depth    uint
+	URL       string
+	Revision  string
+	Path      string
+	Depth     uint
+	SSLVerify bool
```

Comment:
if we could default SSLVerify to true, then we could only provide it when it's false, which would simplify the arguments to this binary and in a lot of test cases :thinking: